app/views/home/index.html.erb

```diff
@@ -0,0 +1,14 @@
+
+<main>
+  <p class='logo'>C<u>ent </u></p>
+  <div class='login_signup'>
+    <p class='session'><%= link_to "Log in", new_user_session_path, class: 'session_text_login' %> </p>
+    <p class='session register'><%= link_to "Sign up", new_user_registration_path, class: 'session_text_signup' %> </p>
+  <div>
+  <div>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+  </div>
+</main>
```

Comment:
- [ ] Kindly consider to place login and sign up buttons on the bottom most part on the screen as visible on the given design as we need to stick to the given design as much as possible.
given design
<img width="293" alt="Screenshot 2023-08-11 at 10 11 23 PM" src="https://github.com/coderacheal/cent/assets/96683843/5b2aae3c-1f02-4774-b5e2-d3933c20e770">